vnext/src/index.uwp.ts

```diff
@@ -9,3 +9,4 @@ export * from './Libraries/Components/Picker/PickerUWP.uwp';
 export * from './Libraries/Components/Popup/Popup.uwp';
 export * from './Libraries/Components/Keyboard/KeyboardExt.uwp';
 export * from './Libraries/Components/Keyboard/KeyboardExtProps';
+export * from './Libraries/Modules/AppTheme/AppTheme.uwp';
```

Comment:
we have the non.uwp files to support typescript builds (which don't know about platform forking).  Not sure if your apptheme will be importable to ts as is.